deploy/nginx.conf

```
@@ -0,0 +1,15 @@
+server {
+  listen 80;
+  location / {
+    proxy_pass http://qpc-server:8000;
+    proxy_set_header    X-Forwarded-Proto $scheme;
```

```suggestion
    proxy_set_header X-Forwarded-Proto $scheme;
```
Incredibly trivial unimportant whitespace fix, but my eyes need consistency! 😂